app/controllers/CreateGroupSelectGroupTypeController.scala

```diff
@@ -72,13 +75,22 @@ class CreateGroupSelectGroupTypeController @Inject()
   }
 
   def showSelectTaxServiceGroupType: Action[AnyContent] = Action.async { implicit request =>
-    Ok(select_group_tax_type(TaxServiceGroupTypeForm.form)).toFuture
+    withGroupNameForAuthorisedOptedAgent { (_, arn) =>
```

Comment:
They won't have a group name at this point right?

Reply:
yeah. i guess. damn